README.md

```diff
@@ -40,8 +44,12 @@ yarn build
 
 ## Running locally
 
+> If you're on macOS and have Podman fully set up, then verify that the initialized virtual machine is currently running. If it's not, then start the machine with `podman machine start`.
```

Comment:
same comment :)

stating the actual command is redundant, as your platform/env may vary.